rust/forevervm/src/main.rs

```diff
@@ -62,6 +64,15 @@ enum MachineCommands {
 async fn main_inner() -> anyhow::Result<()> {
     let cli = Cli::parse();
 
+    env::set_var(
+        "FOREVERVM_RUNNER",
+        env::var("FOREVERVM_RUNNER").unwrap_or_else(|_| "cargo".to_string()),
+    );
+    env::set_var(
+        "FOREVERVM_SDK",
+        env::var("FOREVERVM_SDK").unwrap_or_else(|_| "rust".to_string()),
+    );
```

Comment:
rather than updating the env itself, I would just provide a `get_runner()` function that implements the `env::var("FOREVERVM_RUNNER").unwrap_or_else(|_| "cargo".to_string())` behavior.